logservice.go

```diff
@@ -109,13 +117,19 @@ type app struct {
 	emitterPath,
 	buildID,
 	apiUrl,
-	storeUrl string
+	storeUrl,
+	artifactsLogFile string
```

Comment:
Is it better to rename `buildLogFile` or something?

Reply:
I fixed it.